query-node/mappings/src/proposals.ts

```diff
@@ -323,6 +324,10 @@ async function parseProposalDetails(
     details.frozen = frozen.isTrue
     details.pallet = pallet.toString()
     return details
+  } else if ((proposalDetails as RuntimeProposalDetails_V2002).isSetEraPayoutDampingFactor) {
```

Comment:
Wouldn't the runtime spec version be changed for the Luxor upgrade? So this will be updated.